src/OVAL/results/oval_resultTest.c

```diff
@@ -618,6 +621,13 @@ static oval_result_t eval_check_state(struct oval_test *test, void **args)
 	syschar_model = oval_result_system_get_syschar_model(SYSTEM);
 	ores_clear(&item_ores);
 
+	char *state_names = oval_test_get_state_names(test);
+	dI("To get result of '%s', collected items will be compared with these oval states: %s.",
+		oval_test_get_id(test), state_names);
+	dI("In test '%s' %s of the collected items must satisfy these states: %s.",
+		oval_test_get_id(test), oval_check_get_description(ste_check), state_names);
```

Comment:
I am confused here and I need a little of persuasion here. :-)

Imagine you have a `test` without any `state`s and `OVAL_CHECK_AT_LEAST_ONE`. The resulting sentence will be like `In test 'bla' at least one of the collected items must satisfy these states: '',"` While majority of the people with good engineering training may understand, it will not be 100% clear message. You can get wild philosophical discussion in kitchen about the meaning of `must satisfy none`.

Then imagine next day, you have a `test` without any `state`s and `OVAL_CHECK_NONE_EXIST`. The resulting sentence will be like `In test 'bla' none of the collected items must satisfy these states: '',"` If I apply the same philosophical reasoning as the previous day in kitchen, it works, but I am still confused. Does none satisfy none?

There has to be more convoluted sentences, if you think of the possible combinations. And then, at least one of the convoluted sentences satisfies none. :))

What I want to say is that `satisfy` and `exists` are two different verbs and it would be better for us to have the correct verb in the sense.

Does it make sense to amend the sentences based on the existence of state_names?